src/index.js

```diff
@@ -0,0 +1,29 @@
+import './style.css';
+
+const taskList = document.querySelector('.container2');
+if (taskList) {
+  const tasks = [
+    { description: 'wash the dishes', completed: false, index: 1 },
+    { description: 'complete To Do list', completed: false, index: 2 },
+  ];
+
+  const populateList = () => {
+    tasks
+      .sort((a, b) => a.index - b.index)
+      .forEach((task) => {
+        const listItem = document.createElement('li');
+        const checkbox = document.createElement('input');
+        checkbox.setAttribute('type', 'checkbox');
+        const taskDescription = document.createElement('span');
+        taskDescription.innerHTML = task.description;
+        const taskActions = document.createElement('div');
+        taskActions.innerHTML = '...';
+        listItem.appendChild(checkbox);
+        listItem.appendChild(taskDescription);
+        listItem.appendChild(taskActions);
+        taskList.appendChild(listItem);
+      });
+  };
```

Comment:
- [ ] Kindly align the text of the tasks to the left as we have on the design.


  Your design | Design
  :---:|:---:
  ![yue](https://user-images.githubusercontent.com/54848363/215216190-c2838780-81d4-48ed-ab90-d089de0480fc.jpg) | ![kdkd](https://user-images.githubusercontent.com/54848363/215216196-2c257862-d5be-4cdf-9c27-adb6c95d7b43.jpg)